docs/src/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,214 @@
+NEW (v5.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using the your Afterpay merchant account. This requires a token to be generated by a server to server call to the Afterpay API create checkout endpoint with a paramater of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
+
+{: .info }
+> When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
+
+## Step 1: Install the Cash App PayKit SDK
+
+### Installation (Option One): SPM
+You can install Pay Kit via SPM. Create a new Xcode project and navigate to `File > Swift Packages > Add Package Dependency`. Enter the url `https://github.com/cashapp/cash-app-pay-ios-sdk` and tap **Next**. Choose the `main` branch, and on the next screen, check off the packages as needed.
+
+### Installation (Option Two): Cocoapods
+Add Cocoapods to your to your project. Open the `Podfile` and add `pod 'CashAppPayKit'` and/or `pod 'CashAppPayKitUI'` and save your changes. Run `pod update` and Pay Kit will now be included through Cocoapods
+
+## Step 2: Implement the Pay Kit Observer Protocol
+
+To receive updates from Pay Kit, you’ll need to implement the Pay Kit Observer protocol. Your checkout view controller can conform to this protocol, or you can create a dedicated observer class.
+
+The PayKitObserver protocol contains only one method:
+
+``` swift
+func stateDidChange(to state: CashAppPaySDKState) {
+	// handle state changes
+}
+```
+
+### States
+
+Your implementation should switch on the `state` parameter and handle the appropriate state changes. Some of these possible states are for information only, but most drive the logic of your integration. The most critical states to handle are listed in the table below:
+
+| State | Description |
+|:------|:------------|
+| `ReadyToAuthorize` | Show a Cash App Pay button in your UI and call `authorizeCustomerRequest()` when it is tapped |
+| `Approved` | Grants are ready for your backend to use to create a payment |
+| `Declined` | Customer has declined the Cash App Pay authorization and must start the flow over or choose a new payment method |
+
+### Error States
+
+| State | Description |
+|:------|:------------|
+| `.integrationError` | A fixable bug in your integration |
+| `.apiError` | A degradation of Cash App Pay server APIs. Your app should temporarily hide Cash App Pay functionality |
+| `.integrationError` | A bug outside the normal flow. Report this bug (and what caused it) to Cash App Developer Support |
```

Comment:
How can one distinguish between this `.integrationError` and the other `.integrationError`? Shouldn't this one have a different state code?

Reply:
Copy paste error. Thank you. Addressed in [06867b5](https://github.com/afterpay/sdk-ios/pull/244/commits/06867b5a62ac7fc8895044044309f7a7894aa508)